samples/client/petstore/spring-cloud-no-nullable/src/main/java/org/openapitools/api/PetApi.java

```diff
@@ -1,5 +1,5 @@
 /**
- * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech) (5.3.0-SNAPSHOT).
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech) (5.3.0).
```

Comment:
Can you exclude this file from this PR ?

Reply:
To resolve this, please merge the latest master of https://github.com/OpenAPITools/openapi-generator/ into your branch.